src/binary/ibc-setup/utils/resolve-option.ts

```diff
@@ -1,14 +1,13 @@
 export function resolveOption(
-  identifier: string,
-  ...args: Array<(string | undefined) | (() => string | undefined | void)>
+  ...args: Array<(string | undefined | null) | (() => string | null)>
```

Comment:
`undefined  | null` because it can accept "nothing" values from outside of the program (flags, configs) and values resolved to `null` in the program.

Reply:
```ts
const keyFile = resolveOption(flags.keyFile, process.env.KEY_FILE, app?.keyFile); // can be null
const mnemonic = resolveRequiredOption('mnemonic')(keyFile, ...);
```